components/GameScreen/GameScreen.tsx

```diff
@@ -24,76 +48,186 @@ const GameScreen = ({ onSuccess }: any) => {
-              onClick={() => onClickHandler(option.id)}
-              onMouseEnter={() => {
-                onMouseOverHandler(option.id)
+              }  ${checked[option.id] ? "border-4" : "border-4"} rounded-full disabled:opacity-25 ${
+                loadingAssets && "animate-pulse bg-gray-400 cursor-not-allowed"
+              } w-[100px] h-[100px]`}
+              onClick={() => onClickHandler(option.id, option.musicUrl)}
+              disabled={loadingAssets}
+            >
+              {!loadingAssets && (
+                <Image src={option.imgUrl} alt={option.name} width={100} height={100} />
+              )}
+            </button>
+          </li>
+        ))}
+      </ul>
+      <div className="flex flex-row-reverse gap-4">
+        {chosenAudioTracks.length > 0 &&
+          (!playAudio ? (
+            <button
+              type="button"
+              onClick={() => {
+                setPlayAudio(true)
+                context.resume()
               }}
-              onMouseLeave={() => {
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                fill="none"
+                viewBox="0 0 24 24"
+                strokeWidth={1.5}
+                stroke="currentColor"
+                className="w-6 h-6"
+              >
+                <path
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                  d="M5.25 5.653c0-.856.917-1.398 1.667-.986l11.54 6.348a1.125 1.125 0 010 1.971l-11.54 6.347a1.125 1.125 0 01-1.667-.985V5.653z"
+                />
+              </svg>
+            </button>
+          ) : (
+            <button
+              type="button"
+              onClick={() => {
                 setPlayAudio(false)
+                context.suspend()
               }}
-              disabled={choices.length === 2 && !choices.includes(option.id)}
             >
-              <Image src={option.imgUrl} alt={option.name} width={100} height={100} />
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke-width="1.5"
+                stroke="currentColor"
+                className="w-6 h-6"
+              >
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  d="M15.75 5.25v13.5m-7.5-13.5v13.5"
+                />
+              </svg>
             </button>
-            {playAudio && musicUrl && !checked[option.id] && (
-              <audio autoPlay>
-                <source src={musicUrl} type="audio/mpeg" />
-                <track kind="captions" />
-              </audio>
-            )}
-          </div>
-        ))}
+          ))}
+        {choices.length > 1 && (
+          <MintButton onSuccess={onSuccess} audioTracksToMix={chosenAudioTracks} />
+        )}
```

Comment:
this component is starting to get rather larger, any chance we can break this out into smaller children?